deepchem/feat/tests/test_rdkit_grid_features.py

```diff
@@ -124,16 +128,16 @@ def test_custom_cutoffs(self):
         'cation_pi_dist_cutoff': 5.5,
         'cation_pi_angle_cutoff': 20.0,
     }
-    rgf_featurizer = rgf.RdkitGridFeaturizer(**custom_cutoffs)
+    rgf_featurizer = RdkitGridFeaturizer(**custom_cutoffs)
     self.assertEqual(rgf_featurizer.cutoffs, custom_cutoffs)
 
-  @pytest.mark.skip(reason="TODO: debug")
   def test_rotations(self):
-    featurizer = rgf.RdkitGridFeaturizer(
+    featurizer = RdkitGridFeaturizer(
         nb_rotations=3,
         feature_types=['voxel_combined'],
         flatten=False,
         sanitize=True)
     feature_tensors = featurizer.featurize([(self.ligand_file,
                                              self.protein_file)])
+    print('!!!', feature_tensors.shape)
```

Comment:
Debugging code that got left in?

Reply:
Yes! Thanks for the catch - I'll take that out.